frontend/amp/components/Footer.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { css } from 'react-emotion';
+import { sans } from '@guardian/pasteup/fonts';
+import { palette } from '@guardian/pasteup/palette';
+import InnerContainer from './InnerContainer';
+import { footerLinks, Link } from '../../lib/footerLinks';
+
+const footer = css`
+    background-color: ${palette.neutral[20]};
+    color: ${palette.neutral[86]};
+    font-family: ${sans.body};
+    font-size: 14px;
+`;
+
+const footerInner = css`
+    padding-bottom: 6px;
+`;
+
+const footerLink = css`
+    color: ${palette.neutral[86]};
+    text-decoration: none;
+    padding-bottom: 12px;
+    display: block;
+
+    :hover {
+        text-decoration: underline;
+    }
+`;
+
+const footerList = css`
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    border-top: 1px solid #434343;
```

Comment:
@zeftilldeath what palette colour do we want here?

Reply:
Okay Zef says neutral 46 is a good alternative to the current prod choice so going for that!

Reply:
Anecdotally, it looks pretty similar, just a bit brighter.